dll/win32/shell32/res/icons/Icon_Licence/Tango.txt

```diff
@@ -125,3 +126,5 @@ Please see COPYING for details
 16717.ico
 16718.ico
 16721.ico
+
+http://tango.freedesktop.org/Tango_Icon_Library#Download
```

Comment:
Maybe you can move this DL link at the top of the file where there is the name "Tango project" mentioned.

Reply:
OK.